homeassistant/components/opentherm_gw/__init__.py

```diff
@@ -233,6 +245,20 @@ async def reset_gateway(call):
         DOMAIN, SERVICE_RESET_GATEWAY, reset_gateway, service_reset_schema
     )
 
+    async def set_ch_ovrd(call):
+        """Set the central heating override on the OpenTherm Gateway."""
+        gw_dev = hass.data[DATA_OPENTHERM_GW][DATA_GATEWAYS][call.data[ATTR_GW_ID]]
+        hass.async_create_task(
+            gw_dev.gateway.set_ch_enable_bit(call.data[ATTR_CH_OVRD])
+        )
```

Comment:
```suggestion
        await gw_dev.gateway.set_ch_enable_bit(call.data[ATTR_CH_OVRD])
```

Since its async, can we await it instead?

Reply:
We can, but I'm unsure how Home Assistant handles service calls that take some time to return.
`set_ch_enable_bit` can in some situations take several seconds to return, this is why I chose to use `async_create_task` instead. If this is no problem for Home Assistant (e.g. in the frontend) then we can await here indeed.

Are there any other advantages to `await` over `async_create_task`?

Reply:
If you use `await` any exceptions that bubble up will be reported.    If you use `async_create_task` the error reporting will be far away from the action which will make it hard to track down.

Reply:
Thanks for the info, the change has been made.